packages/angular/src/app/pages/crm-contact-form/contact-form/contact-form.component.html

```diff
@@ -26,156 +27,180 @@
     </dxi-item>
   </dx-toolbar>
 
-  <dx-form
-    *ngIf="!isLoading"
-    class="form last"
-    [formData]="contactData$"
-    labelMode="floating"
-    [readOnly]="!isEditing"
-  >
-    <dxi-item itemType="group" [colCount]="2" caption="Details">
-      <dxi-item dataField="image" template="photo"></dxi-item>
-      <dxi-item itemType="group">
-        <dxi-item
-          dataField="status"
-          [template]="!isEditing ? 'statusField' : 'statusEditing'"
-          [editorOptions]="editorOptions"
+  <div class="form-compact">
+    <dx-form
+      *ngIf="!isLoading"
+      class="form last"
+      [formData]="contactData$"
+      labelMode="floating"
+      [readOnly]="!isEditing"
+    >
+      <dxi-item itemType="group" [colCount]="2">
+        <dxi-item dataField="image" template="photo"></dxi-item>
+        <dxi-item itemType="group">
+          <dxi-item
+            dataField="status"
+            [template]="'statusField'"
+            [editorOptions]="editorOptions"
+          >
+          </dxi-item>
+          <dxi-item
+            dataField="firstName"
+            template="item-plain"
```

Comment:
Here and below in this file uses kebab-case naming of template. But in the whole project uses camelCase.